Sources/GRPCCore/Documentation.docc/Tutorials/Route-Guide/Route-Guide.tutorial

```diff
@@ -93,16 +117,26 @@
 
     @Steps {
       @Step {
-        Create a new empty file in the `Protos` directory called `route_guide.proto`. We'll use
-        the "proto3" syntax and our service will be part of the "routeguide" package.
+        Create a new directory in the `Sources/Protos` directory called `routeguide`
+        using `mkdir Sources/Protos/routeguide`.
+      }
+
+      @Step {
+        Create a new empty file in the `Sources/Protos/routeguide` directory
+        called `route_guide.proto`. We'll use the "proto3" syntax and our service will be part of
+        the "routeguide" package.
 
-        @Code(name: "Protos/route_guide.proto", file: "route-guide-sec02-step01-import.proto")
+        It's good practice to organise your `.proto` files according to the package they are
```

Comment:
Nit: should we stick to American spelling (organise vs organize)?

Reply:
We should. (Sadly.)